doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -35,15 +39,19 @@ This example shows how to launch Fluent in solution mode:
 
```

Comment:
We can add a note here that if no mode is passed as argument then by default a solver session is launched. This can either be added here or as a note in the first paragraph. Then all other suggestions will make sense. Please let me know what you think @seanpearsonuk.
Thank you.

Reply:
We will do this in a following PR